compiler/rustc_parse/messages.ftl

```diff
@@ -364,6 +364,10 @@ parse_inner_doc_comment_not_permitted = expected outer doc comment
     .label_does_not_annotate_this = the inner doc comment doesn't annotate this {$item}
     .sugg_change_inner_to_outer = to annotate the {$item}, change the doc comment from inner to outer style
 
+parse_invalid_attr_unsafe = `{$name}` is not an unsafe attribute
+    .suggestion = remove the `unsafe(...)`
+    .note = extraneous unsafe is not allowed in attributes
```

Comment:
Discussion: should this say something like "unsafe attribute block is not permitted for safe attributes"?

Reply:
I'm fine with this wording for now, we can always adjust it if we come up with better wording.